deploy/foxlake-cluster/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
ClusterRoleBinding is necessary because foxlake needs to be granted cluster-wide access permissions. I have tested with RoleBinding, the `create cluster` function can not work properly